src/GitHub.VisualStudio.UI/RepositoryOrigin.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace GitHub.VisualStudio.UI
+{
+    public enum RepositoryOrigin
+    {
+        DotCom,
+        Enterprise,
+        Other,
+        NonGitRepository,
+    }
```

Comment:
Well strictly speaking, the `Unknown` state is `RepositoryOrigin == null` as the `RepositoryOrigin` property is currently nullable, because it came from the nullable `IsGitHubRepo` property. However I agree that it would be better the have that `Unknown` value as an enum value rather than the null state: will change.